<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dst_window</class>
 <widget class="QWidget" name="dst_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Пароли dst</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="dst_find_input">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите наименование dst-файла</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dst_find_button">
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Парольная фраза</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="dst_pass_browser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
